Ginger/GingerCoreNET/ActionsLib/UI/VisualTesting/ActVisualTesting.cs

```diff
@@ -508,7 +508,15 @@ private void Compare()
             else
             {
                 // get it from the file
-                string fullTargetFilePath = GetFullFilePath(TargetFileName);
+                string targetFileName = GetInputParamCalculatedValue(Fields.SavedTargetImageFilenameString);
```

Comment:
Previously we were sending TargetFileName, now its changed to Fields.SavedTargetImageFilenameString. issue was due to that or due to value expression was not getting calculated

Reply:
issue is related to value expression not getting calculated